pom.xml

```diff
@@ -493,6 +494,16 @@
         <artifactId>chill-java</artifactId>
         <version>${chill.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.esotericsoftware</groupId>
+        <artifactId>kryo-shaded</artifactId>
+        <version>${kryo.version}</version>
+      </dependency>
+      <dependency>
```

Comment:
Kryo 4.0.x depends on version objenesis 2.5.1 , while Spark currently depends on objenesis 3.3. Here, it is uniformly defined as version 3.3.

Reply:
Shall we use exclusion ?

Reply:
I don't think it's necessary to use `exclusion`. But just to be on the safe side, let me exclude it

Reply:
done